newsroom/utils.py

```diff
@@ -149,27 +152,59 @@ async def get_json_or_400_async(req: Request):
     return data
 
 
-def success_response(data: Any, headers: Sequence = ()):
+def success_response(data: Any, headers: Sequence = (), status_code: int = 200):
     """
     Shortcut to return a `superdesk.core.web.Response` with a
     status_code 200 and empty headers by default
     """
     if isinstance(data, BaseModel):
         data = data.model_dump(by_alias=True, exclude_unset=True)
 
-    return Response(data, 200, headers)
+    return Response(data, status_code, headers)
```

Comment:
I'm wondering if instead we make the `headers` and `status_code` attributes for the `Response` class in superdesk-core optional?

Then we can simply use `return Response({"success": True})`, and in [superdesk-core -> app](https://github.com/superdesk/superdesk-core/blob/async/superdesk/factory/app.py#L259) we can test if `isinstance(response.body, BaseModel)` and support that in core (as we'll need that for Superdesk as well).

This will improve the core framework, making it easier to work with for all apps that will use it

Reply:
Yeah, I agree with that. I'll make the PR for core shortly.